lib/materialist/workers/event.rb

```diff
@@ -38,6 +39,11 @@ def report_stats(topic, action, kind)
           tags: ["action:#{action}", "topic:#{topic}"]
         )
       end
+
+      def notice_error(exception, event)
+        return unless handler = Materialist.configuration.notice_error
```

Comment:
Or to the comment above, maybe do `raise unless handler = Materialist.configuration.notice_error`. Then you can remove the explicit `raise` above.

Probably I'd change the name of the method to `handle_error` to make it clear.